src/main/java/com/eternalcode/combat/fight/controller/FightTagController.java

```diff
@@ -34,6 +35,13 @@ void onEntityDamageByEntity(EntityDamageByEntityEvent event) {
             return;
         }
 
+        List<EntityType> entityTypes = this.config.settings.entityTypesToLog;
+        EntityType damagerType = event.getDamager().getType();
+
+        if (!entityTypes.contains(damagerType)) {
+            return;
+        }
```

Comment:
Teraz można sprawdzić czy dany pocisk jest na blacklist i w razie czego zreturnować metodę 

```suggestion
        List<EntityType> disabledProjectileEntites = this.config.settings.disabledProjectileEntites;
        
        if (event.getDamager() instanceof Projectile projectile && disabledProjectileEntites.contains(projectile.getType())) {
            return;
        }
```